daml-lf/integration-test-lib/src/main/com/daml/lf/CantonFixture.scala

```diff
@@ -71,6 +71,7 @@ trait CantonFixture
   protected lazy val nParticipants: Int = 1
   protected lazy val timeProviderType: TimeProviderType = TimeProviderType.WallClock
   protected lazy val tlsEnable: Boolean = false
+  protected lazy val applicationId: ApplicationId = ApplicationId(getClass.getName)
```

Comment:
Q: do we want to allow extending classes to alter the application ID? IIRC, isn't this used for (default) authentication?

Reply:
Offline chats indicate that we should keep this value as is.